code/modules/clothing/gloves/special.dm

```diff
@@ -50,6 +50,13 @@
 
 	QDEL_NULL(pull_component_weakref)
 
+/obj/item/clothing/gloves/supermatter_gloves
```

Comment:
Changing the path to `/obj/item/clothing/gloves/noblium_gloves` would make it clearer that this isn't a "Slap people into dust" gloves.

Reply:
changed